ddmrp_adjustment/wizards/ddmrp_adjustment_wizard_view.xml

```diff
@@ -26,22 +21,31 @@
                         </tree>
                     </field>
                 </group>
+                <group name="factor" string="Factors to Apply">
+                    <field name="apply_daf"/>
+                    <field name="apply_ltaf"/>
+                </group>
                 <group name="factor_sheet"
                        string="Sheet">
                     <p>Fill the factors to apply in each period for all the
                        selected Buffers.</p><br></br>
-                    <field name="line_ids" nolabel="1" colspan="4"
-                           widget="x2many_2d_matrix"
```

Comment:
Could you read the matrix functionality, the widget is already merged in OCA

Reply:
@lreficent I tried to use it but it seems there's still a bug as it doesn't appear at the moment because values are added in an onchange.
So we tried to fix it but aren't there yet...
I'll ping you when last changes will be done.

Reply:
@grindtildeath that sounds great, thanks!

Reply:
@lreficent Just opened https://github.com/OCA/web/pull/915 to fix the display of x2m matrix widget.
Can you try it out before I change anything here ?